ipatests/test_integration/test_installation.py

```diff
@@ -703,6 +703,30 @@ def install(cls, mh):
     def test_install_master(self):
         tasks.install_master(self.master, setup_dns=False)
 
+    def test_smoke_test_for_debug_mode(self):
+        """Test if an IPA server works in debug mode.
+        Related: https://pagure.io/freeipa/issue/8891
+        """
+
+        target_fname = paths.IPA_SERVER_CONF
+        assert not self.master.transport.file_exists(target_fname)
```

Comment:
Is it expected that this test fails if `server.conf` already exists? In future this assumption may be broken.

Reply:
Yes, this is expected, so that we can adapt the test.

Reply:
May be adapt without waiting for breakage? For example, check for existance and edit with `FileBackup` if exists.